Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/DivXmlElementAccess.java

```diff
@@ -105,9 +110,17 @@ public DivXmlElementAccess() {
             super.getMetadata().addAll(readMetadata((MdSecType) mdSecType, amdSecTypeOf(mets, (MdSecType) mdSecType)));
         }
         metsReferrerId = div.getID();
+        BigInteger order = div.getORDER();
+        if (Objects.nonNull(order) && order.intValue() > 0) {
+            super.setOrder(order.intValue());
```

Comment:
```suggestion
            setOrder(order.intValue());
```

Reply:
You don't need `super `here. As far as I see thats counts for every `super` in this class. Remove at least those you added ;-)